packages/cli/test/unit/License.test.ts

```diff
@@ -252,4 +252,16 @@ describe('License', () => {
 			});
 		});
 	});
+
```

Comment:
for completeness, would be good to have a test of `init called with recreate true`

Reply:
Added

Reply:
Sorry I meant a separate unit test for `init` called with `recreate = true`

```
it('initializes again if recreate is true', () => {
  ...
  license.init('main', true);
  ...
```